lms/djangoapps/instructor/views/api.py

```diff
@@ -2707,10 +2707,129 @@ def start_certificate_regeneration(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_global_staff
+@require_http_methods(['POST', 'DELETE'])
+def certificate_exception_view(request, course_id, certificate_exception_id=None):
```

Comment:
I think instead of passing `certificate_exception_id` as second argument here we should get user like this
`certificate_exception.get('user_name', '') or certificate_exception.get('user_email', '')` as we are doing in `add_certificate_exception` method and remove that user's entry from whitelist model for given course. That way this API would be consistent with PUT/POST scenarios.